CMakeLists.txt

```diff
@@ -608,7 +608,6 @@ endif(FEATURE_STANDALONE_GC)
 
 if (CLR_CMAKE_PLATFORM_UNIX)
   include_directories("src/pal/inc")
-  include_directories("src/pal/inc/rt")
```

Comment:
Could you please leave the include directory here and move the block at lines 624..634 to line 589 before the clrdefinitions.cmake is included and remove the `remove_definitions` commands from that block? The test binaries are completely independent of the PAL and definitions that we define for the runtime, they even have their own header with cross platform stuff definitions.

Reply:
Will do, thanks.